### PR TITLE
Add flip to the match options on the edit image template view

### LINKED ISSUE #124: Add flip option to the edit template view
Possible values are: `'horizontal', 'h', 'vertical', 'v', 'both', 'b', 'all', 'a'` according to the CLI, so this could reasonably be a dropdown.

### PR DESCRIPTION
**Pull request by csbailey5t**
Fixes #124

### CODE REVIEW

File: app/assets/javascripts/components/flip_form.es6.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+FlipForm.propTypes = {
+  object: React.PropTypes.string.isRequired,
+  attribute: React.PropTypes.string.isRequired,
+  type: React.PropTypes.string.isRequired,
+  value: React.PropTypes.number,
+  handleEnableChange: React.PropTypes.func,
+};
+
+FlipForm.defaultProps = {
+  min: 0,
```
Comment (mejackreed) [inline app/assets/javascripts/components/flip_form.es6.jsx]: Are `min`, `max` needed? Should default `value` be set as something else?

**Review by mejackreed — CHANGES_REQUESTED** [review]
Just that one minor change, all looks great though!

File: app/assets/javascripts/components/flip_form.es6.jsx (hunk 1 above)
Comment (csbailey5t) [reply app/assets/javascripts/components/flip_form.es6.jsx]: Those were leftovers from the form I based this on - I've removed them as well as a value in the initial state that was no longer necessary.

Comment (mejackreed) [reply app/assets/javascripts/components/flip_form.es6.jsx]: 👍

**Review by mejackreed — APPROVED** [review]

**Pull request merged**